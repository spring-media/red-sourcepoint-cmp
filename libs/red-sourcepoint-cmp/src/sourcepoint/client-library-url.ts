export const getClientLibraryUrl = (): string => `https://cmp2.bild.de/wrapperMessagingWithoutDetection.js`;
