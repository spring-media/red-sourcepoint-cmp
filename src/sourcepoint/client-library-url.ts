export const getClientLibraryUrl = (): string => `https://cdn.privacy-mgmt.com/wrapperMessagingWithoutDetection.js`;
