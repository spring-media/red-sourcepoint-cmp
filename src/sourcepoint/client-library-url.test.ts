import { getClientLibraryUrl } from './client-library-url';

describe('sourcepoint getClientLibraryUrl', () => {
  it('should return the expected content', () => {
    expect(getClientLibraryUrl()).toMatchSnapshot();
  });
});
