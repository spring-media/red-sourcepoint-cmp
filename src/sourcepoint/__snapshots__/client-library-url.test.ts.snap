// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sourcepoint getClientLibraryUrl should return the expected content 1`] = `"https://gdpr-tcfv2.sp-prod.net/wrapperMessagingWithoutDetection.js"`;
